book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [x] Please make sure in `Book` class a method for adding rentals should be present and make sure the rentals variable is also writable so move it to `attr_accessor`.
  - You can consider the below code

```ruby
 def add_rental(person, date)
    Rental.new(date, self, person)
  end
```